.circleci/config.yml

```diff
@@ -77,13 +86,13 @@ jobs:
             at: artifacts
       - run:
           name: "Decode signing cert"
-          command: base64 -D -o cert.p12 \<<< $SIGNING_CERT
+          command: base64 -D -o cert.p12 \<<< $SIGNING_CERT_TEMP
```

Comment:
Putting in temp vars to point at a new account, I can't get the company cert to work.